components/bio-formats-plugins/src/loci/plugins/out/Exporter.java

```diff
@@ -153,15 +176,36 @@ public void run() {
                 if (save != null) outfile = save;
             }
         }
-
+        //create a temporary file if window less
+        if (windowless && (outfile == null || outfile.length() == 0)) {
+            try {
+                String name = removeExtension(imp.getTitle());
+                String n = name+ ".ome.tif";
+                File tmp = File.createTempFile(name, ".ome.tif");
+                File p = tmp.getParentFile();
```

Comment:
This results in an extra empty file being present in /tmp - could this either be deleted upon exit, or replaced with ```System.getProperty("java.io.tmpdir")```?